ui-v2/src/components/concurrency/concurrency-page.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+import { ConcurrencyPageTitle } from "./concurrency-page-title";
+import { ConcurrencyTabs } from "./concurrency-tabs";
+import { GlobalConcurrencyView } from "./global-concurrency-view";
+import { TaskRunConcurrencyView } from "./task-run-concurrenct-view";
+
+type Tab = "Global" | "Task Run";
```

Comment:
This looks like it's duplicative of `Tabs` in `ui-v2/src/components/concurrency/concurrency-tabs.tsx`. I think it makes sense to keep `Tabs` an import it into this module.